test/test_helper.rb

```diff
@@ -2,6 +2,9 @@
 inc_dirs = Dir.glob("#{root_location}/*/src")
 ENV["Y2DIR"] = inc_dirs.join(":")
 
+ENV["LANG"] = "C"
```

Comment:
I'd prefer `en_US.UTF-8` I think there could be possibly some problems with UTF-8 strings. Moreover using non-UTF8  is probably not supported and should not be normally used in the target systems.

Reply:
Right, lets use that.